### PR TITLE
Desktop Responsive Portfolio

### PR DESCRIPTION
**Pull request by ridaarif98**
I made my portfolio desktop responsive. 
Add media queries in the stylesheet to make it responsive for different screen sizes.
Update README file with more detail of portfolio.
Adjust spaces.
Integrate linter
Resolve linter errors

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -213,17 +223,23 @@ <h1>About me</h1>
...
     <input type="text" name="user_name" placeholder="Full Name" maxlength="30" required>
     <input type="email" name="user_email" placeholder="Email address" required>
-    <textarea name="user_text" maxlength="500"  required>Enter text here</textarea>
+    <textarea name="user_text" maxlength="500" placeholder="Enter the text here"  required></textarea>
     <button type="submit" class="contactBtn">Get in Touch</button>
   </form>
+</div>
+<div class="line">
+<hr class="responsiveHr">
+</div>
```
Comment (ElijahTobs) [inline Index.html:240]: - [ ] This `hr` tag is not visible on your page. The grey line should be at the top of the social media icons.

Yours | Original
----- | -----
image | image

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi Team 👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional and you've followed the Github flow rules. ✔️ 
You have good commit messages ✔️ 
Good Pull Request (PR) title and summary ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

File: Index.html [hunk 2]
```diff
@@ -14,11 +14,19 @@
...
+  
   <a href="#"><h1>Welcome</h1></a>
   <nav class="nav-bar">
+    <ul>
+      <li><a href="#">Portfolio</a> </li>
+      <li> <a href="#">About</a> </li>
+      <li> <a href="#">Contact</a> </li>
+    </ul> 
     <i class="fa fa-bars" ></i>
   </nav>
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply Index.html]: Well addressed this👍

File: style.css [hunk 3]
```diff
@@ -90,6 +147,99 @@ body {
...
+    height: 90vh;
+  }
+
+  .headline .section-heading {
+    padding-top: 160px;
+    font-weight: 800;
+    padding-left: 200px;
+    padding-right: 200px;
+    padding-bottom: 26px;
+    line-height: 60px;
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply style.css]: Well resolved this👍

File: style.css [hunk 4]
```diff
@@ -139,12 +289,70 @@ body {
...
+
+  .Work hr {
+    display: none;
+  }
+
+  .flex-box {
+    float: left;
+    margin-right: 2%;
+    width: 47%;
+  }
```
[the remark this answers is not in the document]
Comment (DeliceLydia) [reply style.css]: Addressed this👍

File: Index.html (hunk 1 above)
Comment (DeliceLydia) [reply Index.html]: This `hr` still is not visible work on that and you will b good to go

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @ridaarif98😊, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments I added under the previous review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 1 above)
Comment (ridaarif98) [reply Index.html]: Thanks for your review. But on my side line is displaying
ss8

Comment (DeliceLydia) [reply Index.html]: Oooh sorry I made a mistake I checked social media icons for the main section
ask another CR your project will be approved

I apologize for the inconvenience!

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi @ridaarif98, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**